res/res_respoke/respoke_sdk_header.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Respoke - Web communications made easy
+ *
+ * Copyright (C) 2014, D.C.S. LLC
+ *
+ * Chad McElligott <cmcelligott@digium.com> 
+ *
+ * See https://www.respoke.io for more information about
+ * Respoke. Please do not directly contact any of the
+ * maintainers of this project for assistance.
+ * Respoke offers a community forum to submit and discuss
+ * issues at http://community.respoke.io; please raise any
+ * issues there.
+ *
+ * See http://www.asterisk.org for more information about
+ * the Asterisk project. Please do not directly contact
+ * any of the maintainers of this project for assistance;
+ * the project provides a web site, mailing lists and IRC
+ * channels for your use.
+ *
+ * This program is free software, distributed under the terms of
+ * the GNU General Public License Version 2. See the LICENSE file
+ * at the top of the source tree.
+*/
+#include <stddef.h>
+#include <sys/utsname.h>
+#include <stdio.h>
+#include "asterisk/ast_version.h"
+#include "include/respoke_version.h"
+
+int respoke_get_sdk_header(char *buf, size_t len) {
```

Comment:
Since you don't check the return code, and it really can't fail anyways, you can return `void` instead.